app/jobs/regular/update_post_uploads_secure_status.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Jobs
+  class UpdatePostUploadsSecureStatus < ::Jobs::Base
```

Comment:
Since we're updating the source, maybe the job should be called `UpdatePostUploadsSecureSource`?